lib/sedna/README.md

```diff
@@ -1,2 +1,289 @@
 
-### Sedna Python SDK
+# Sedna Python SDK
+
+The Sedna Python Software Development Kit (SDK) aims to provide developers with a convenient yet flexible tool to write the Sedna applications.
+
+This document introduces how to obtain and call Sedna Python SDK.
+
+
+## Introduction
+
+Expose the Edge AI features to applications (i.e., training or inference programs).
+
+## Requirements and Installation
+The build process is tested with Python 3.6, Ubuntu 18.04.5 LTS.
+
+```bash
+# Clone the repo
+git clone --recursive git@github.com:kubeedge/sedna.git
```

Comment:
https instead ssh protocol for git-clone

Reply:
check